packages/vuetify/src/util/helpers.ts

```diff
@@ -413,17 +413,21 @@ export function sortItems (
   })
 }
 
+export type filterFn = (value: any, search: string, item: any) => boolean
```

Comment:
Types in PascalCase:

```suggestion
export type FilterFn = (value: any, search: string, item: any) => boolean
```